Assets/MixedRealityToolkit/_Core/Definitions/InputSystem/MixedRealitySpeechCommandsProfile.cs

```diff
@@ -12,6 +14,35 @@ namespace Microsoft.MixedReality.Toolkit.Core.Definitions.InputSystem
     [CreateAssetMenu(menuName = "Mixed Reality Toolkit/Mixed Reality Speech Commands Profile", fileName = "MixedRealitySpeechCommandsProfile", order = (int)CreateProfileMenuItemIndices.Speech)]
     public class MixedRealitySpeechCommandsProfile : ScriptableObject
     {
+        [SerializeField]
+        [Tooltip("Speech System Class to instantiate at runtime.")]
+        [Implements(typeof(IMixedRealitySpeechSystem), TypeGrouping.ByNamespaceFlat)]
+        private SystemType speechSystemType = null;
+
+        /// <summary>
+        /// Speech System Script File to instantiate at runtime.
+        /// </summary>
+        public SystemType SpeechSystemType => speechSystemType;
+
+        [SerializeField]
+        [Tooltip("Whether the recognizer should be activated on start.")]
+        private AutoStartBehavior startBehavior = AutoStartBehavior.AutoStart;
+
+        /// <summary>
+        /// The list of Speech Commands users use in your application.
+        /// </summary>
+        public AutoStartBehavior SpeechRecognizerStartBehavior => startBehavior;
+
+        [SerializeField]
+        [Tooltip("0 == High, 1 == Medium, 2 == Low, 3 == Unknown")]
+        private int recognitionConfidenceLevel = 1;
```

Comment:
why isn't this an enum?

Reply:
Bc that enum is platform specific and this way we can serialize the data even if we switch the platform.

Also for other platforms it may have a different enum structure.  I'm open to making our own clone enum for this.

Reply:
Yes, let's please define our own clone enum. That way, we can keep the easy cross plat promise.

Reply:
Done